CudaFunctions.h

```diff
@@ -4,14 +4,14 @@
 #ifdef CUDA
 #include "hapi.h"
 #include "HostCUDA.h"
-void TreePieceCellListDataTransferBasic(CudaRequest *data, hapiWorkRequest *wr);
-void TreePiecePartListDataTransferBasic(CudaRequest *data, hapiWorkRequest *wr);
+void TreePieceDataTransferBasic(CudaRequest *data, CudaDevPtr *ptr);
```

Comment:
Looks like I missed this one. It sounds like you prefer the docstrings to go in the .h files (rather than above the function definitions in the .cpp and .cu files). Should I move the other docstrings I created in HostCUDA.cu to the header files?

Reply:
Following the old convention, I added the docstrings to HostCUDA.cu. Let me know if you think it would be better to add the documentation for the HostCUDA functions to the respective .h files instead.